### PR TITLE
Fix typo in add-k8s

### PR DESCRIPTION
**Pull request by wallyworld**
Fix typo in juju add-k8s

### CODE REVIEW

**Comment by wallyworld** [comment]
$$merge$$

File: cmd/juju/caas/add.go [hunk 1]
```diff
@@ -352,7 +352,6 @@ func (c *AddCAASCommand) Run(ctx *cmd.Context) error {
 	if c.hostCloudRegion == "" {
 		return errors.New(clusterQueryErrMsg)
 	}
-	newCloud.HostCloudRegion = c.hostCloudRegion
```
Comment (howbazaar) [inline cmd/juju/caas/add.go]: This is not a typo? Perhaps a pasteo?

**Comment by jameinel** [comment]
This looks like a real test failure
```
----------------------------------------------------------------------
FAIL: add_test.go:683: addCAASSuite.TestCorrectParseFromStdIn

[LOG] 0:00.000 DEBUG  Call: PublicCloudMetadata([[]])
[LOG] 0:00.000 DEBUG  Results: [map[mrcloud1:{mrcloud1 kubernetes   []    [] map[] map[] []} mrcloud2:{mrcloud2 kubernetes   []    [] map[] map[] []}] false <nil>]
[LOG] 0:00.001 DEBUG juju.caas.kubernetes.clientconfig no cluster name specified, so use current context "the-context"
[LOG] 0:00.001 DEBUG juju.caas.kubernetes.clientconfig name "the-user", user &api.AuthInfo{LocationOfOrigin:"", ClientCertificate:"", ClientCertificateData:[]uint8(nil), ClientKey:"", ClientKeyData:[]uint8(nil), Token:"", TokenFile:"", Impersonate:"", ImpersonateGroups:[]string(nil), ImpersonateUserExtra:map[string][]string(nil), Username:"theuser", Password:"thepassword", AuthProvider:(*api.AuthProviderConfig)(nil), Exec:(*api.ExecConfig)(nil), Extensions:map[string]runtime.Object{}}
[LOG] 0:00.001 DEBUG juju.cmd.juju.k8s caasConfig: &{Type:kubernetes Contexts:map[the-context:{CloudName:the-cluster CredentialName:the-user}] CurrentContext:the-context Clouds:map[the-cluster:{Endpoint:https://1.1.1.1:8888 Attributes:map[CAData:A]}] Credentials:map[the-user:{authType:userpass attributes:map[username:theuser password:thepassword] Revoked:false Label:kubernetes credential "the-user" Invalid:false InvalidReason:}]}
[LOG] 0:00.001 DEBUG  Call: GetClusterMetadata([])
[LOG] 0:00.001 DEBUG  Results: [0xc0006b67f0 <nil>]
[LOG] 0:00.001 DEBUG  Call: CheckDefaultWorkloadStorage([])
[LOG] 0:00.001 DEBUG  Results: [<nil>]
[LOG] 0:00.001 DEBUG  Call: PersonalCloudMetadata([])
[LOG] 0:00.001 DEBUG  Results: [map[mrcloud2:{mrcloud2 kubernetes   []    [] map[] map[] []} mrcloud1:{mrcloud1 kubernetes   []    [] map[] map[] []}] <nil>]
[LOG] 0:00.001 DEBUG  Call: WritePersonalCloudMetadata([map[mrcloud1:{mrcloud1 kubernetes   []    [] map[] map[] []} mrcloud2:{mrcloud2 kubernetes   []    [] map[] map[] []} myk8s:{myk8s kubernetes   [userpass] https://1.1.1.1:8888   [] map[] map[] [A]}]])
[LOG] 0:00.001 DEBUG  Results: [<nil>]
[LOG] 0:00.001 DEBUG  Call: AddCloud([{myk8s kubernetes   [userpass] https://1.1.1.1:8888   [] map[] map[] [A]}])
[LOG] 0:00.001 DEBUG  Results: []
add_test.go:690:
    s.assertStoreClouds(c, "gce/us-east1")
/home/ubuntu/go/src/github.com/juju/juju/vendor/github.com/juju/testing/stub.go:239:
    c.Check(call, jc.DeepEquals, expected)
... obtained testing.StubCall = testing.StubCall{FuncName:"WritePersonalCloudMetadata", Args:[]interface {}{map[string]cloud.Cloud{"mrcloud1":cloud.Cloud{Name:"mrcloud1", Type:"kubernetes", HostCloudRegion:"", Description:"", AuthTypes:cloud.AuthTypes(nil), Endpoint:"", IdentityEndpoint:"", StorageEndpoint:"", Regions:[]cloud.Region(nil), Config:map[string]interface {}(nil), RegionConfig:cloud.RegionConfig(nil), CACertificates:[]string(nil)}, "mrcloud2":cloud.Cloud{Name:"mrcloud2", Type:"kubernetes", HostCloudRegion:"", Description:"", AuthTypes:cloud.AuthTypes(nil), Endpoint:"", IdentityEndpoint:"", StorageEndpoint:"", Regions:[]cloud.Region(nil), Config:map[string]interface {}(nil), RegionConfig:cloud.RegionConfig(nil), CACertificates:[]string(nil)}, "myk8s":cloud.Cloud{Name:"myk8s", Type:"kubernetes", HostCloudRegion:"", Description:"", AuthTypes:cloud.AuthTypes{"userpass"}, Endpoint:"https://1.1.1.1:8888", IdentityEndpoint:"", StorageEndpoint:"", Regions:[]cloud.Region(nil), Config:map[string]interface {}(nil), RegionConfig:cloud.RegionConfig(nil), CACertificates:[]string{"A"}}}}}
... expected testing.StubCall = testing.StubCall{FuncName:"WritePersonalCloudMetadata", Args:[]interface {}{map[string]cloud.Cloud{"mrcloud1":cloud.Cloud{Name:"mrcloud1", Type:"kubernetes", HostCloudRegion:"", Description:"", AuthTypes:cloud.AuthTypes(nil), Endpoint:"", IdentityEndpoint:"", StorageEndpoint:"", Regions:[]cloud.Region(nil), Config:map[string]interface {}(nil), RegionConfig:cloud.RegionConfig(nil), CACertificates:[]string(nil)}, "mrcloud2":cloud.Cloud{Name:"mrcloud2", Type:"kubernetes", HostCloudRegion:"", Description:"", AuthTypes:cloud.AuthTypes(nil), Endpoint:"", IdentityEndpoint:"", StorageEndpoint:"", Regions:[]cloud.Region(nil), Config:map[string]interface {}(nil), RegionConfig:cloud.RegionConfig(nil), CACertificates:[]string(nil)}, "myk8s":cloud.Cloud{Name:"myk8s", Type:"kubernetes", HostCloudRegion:"gce/us-east1", Description:"", AuthTypes:cloud.AuthTypes{"userpass"}, Endpoint:"https://1.1.1.1:8888", IdentityEndpoint:"", StorageEndpoint:"", Regions:[]cloud.Region(nil), Config:map[string]interface {}(nil), RegionConfig:cloud.RegionConfig(nil), CACertificates:[]string{"A"}}}}}
... mismatch at .Args[0]["myk8s"].HostCloudRegion: unequal; obtained ""; expected "gce/us-east1"
```

**Comment by wallyworld** [comment]
$$merge$$

**Comment by wallyworld** [comment]
$$merge$$

**Pull request merged**